build.gradle

```diff
@@ -1,8 +1,9 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
   repositories {
-    mavenCentral()
     jcenter()
+    maven { url "https://jitpack.io" }
```

Comment:
https://github.com/javiersantos/MaterialStyledDialogs, but this change shouldn't have been in here as it's already included in `app/build.gradle`. Should have waited for a rereview.